Owncloud iOs Client/AppDelegate.m

```diff
@@ -135,16 +135,18 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     
     //Configuration UINavigation Bar apperance
     [self setUINavigationBarApperanceForNativeMail];
-    
-    //Init and update the DataBase
-    [InitializeDatabase initDataBase];
+
+    [self showSplashScreenFake];
```

Comment:
Why do we have a fake splash screen?

Reply:
Legacy.. should be needed to show meanwhile and the name maybe not the most accuracy

Reply:
so maybe change the name? 🙂  XCode has good tools for refactoring Objective-C code. 😉
And also meanwhile what? are you doing some checks meanwhile the "fake screen" is on the screen? 🤨 
This doesn't look good from a UI perspective because the cloud icon in the the real splash screen doesn't match with the one in the "fake" splash screen  in height and high

Reply:
👍 This is not related to this pr, but since it just really tiny we can change it now. We have to consider that refactor of no related code could increment QA time.

Reply:
And sadly.. Xcode did not have a great refactor tools xD, they have improved a lot with latest version. Yeah!

Reply:
>This doesn't look good from a UI perspective because the cloud icon in the the real splash screen doesn't match with the one in the "fake" splash screen in height and high

if this happens should be consider in other issue